### DISCUSSION TITLE
📣 New Release: Splunk Enterprise Security 7.1

### DISCUSSION THREAD

**Discussion by ZachTheSplunker** [splunk/rba#39] [Announcements]
## Highlighted Features of this Release

- (RBA Visualization) Threat Topology
- (RBA Visualization) MITRE ATT&CK Framework Matrix
- Normalized Risk Object Field
- and more! -> Check out the [release notes][release-notes]!

Splunkbase: https://splunkbase.splunk.com/app/263

### Threat Topology

See the scope of the incident beyond the infected user, providing improved situational awareness, an expanded viewpoint, and a comprehensive view across the SOC team.

![Threat Topology](https://user-images.githubusercontent.com/56560139/211844343-7d944e88-8729-45dd-bc43-6e275f7c78d3.png)

### MITRE ATT&CK Framework Matrix

Security analysts can quickly build situational awareness around an incident in the context of the MITRE ATT&CK Matrix and pivot directly to associated MITRE documentation.

![MITRE Matrix](https://user-images.githubusercontent.com/56560139/211843902-1787cb83-0187-405e-a692-0ba6b7ace7e2.png)

### Normalized Risk Object field

Previously, RBA failed to account for Assets & Identities (A&I). Risk events were grouped strictly by Risk Object. If multiple risk events contained different risk objects, they would _**not**_ be grouped together, even if the risk objects referred to the _same_ Asset or Identity.

There is now a `normalized_risk_object` field in the Risk Data Model, which accounts for A&I data. Risk notables are grouped by this field allowing risk objects to be associated with an Asset or Identity.

Find out more on [Splunk Docs](https://docs.splunk.com/Documentation/ES/7.1.0/RBA/ReviewRiskNotables#Display_risk_notables_generated_from_the_same_risk_object)

### [Release notes][release-notes]

See all the release notes at [https://docs.splunk.com/Documentation/ES/7.1.0/RN/Enhancements][release-notes]

[release-notes]: https://docs.splunk.com/Documentation/ES/7.1.0/RN/Enhancements#What.27s_new

**Comment by matt-snyder-stuff** [comment]
😍that Threat topology!!!!
